app/event/raise-event.js

```diff
@@ -0,0 +1,28 @@
+const { PublishEvent } = require('ffc-ahwr-event-publisher')
+const { eventQueue } = require('../config').mqConfig
+
+const raiseEvent = async (event, status = 'success') => {
+  const eventPublisher = new PublishEvent(eventQueue)
+
+  const eventMessage = {
+    name: event.name,
+    properties: {
+      id: event.id,
+      sbi: event.sbi,
+      cph: event.cph,
+      checkpoint: process.env.APPINSIGHTS_CLOUDROLE,
+      status,
+      ip: event.ip,
```

Comment:
Just a note to check on whether we can handle the IP address from a compliance/security point of view in relation to the other PR comment -> https://github.com/DEFRA/ffc-ahwr-event/pull/5/files